packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.mjml

```diff
@@ -1,33 +1,44 @@
-<%# This Source Code Form is subject to the terms of the Mozilla Public
-  # License, v. 2.0. If a copy of the MPL was not distributed with this
+<%# This Source Code Form is subject to the terms of the Mozilla Public 
+  # License, v. 2.0. If a copy of the MPL was not distributed with this 
   # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
 
 <mj-include path="./css/appBadges/index.css" type="css" css-inline="inline" />
 
+<%# TODO: Conslidate app store link names %>
+<% locals.iosUrl = locals.iosUrl || locals.appStoreLink %> 
+<% locals.androidUrl = locals.androidUrl ||  locals.playStoreLink %> 
```

Comment:
Yeah I am now no longer confident that we will always have both of these when users download a subscription product. 😅 I think it's all fine to leave the variables like this (but you could do `iosUrl = locals.iosUrl | locals.appStoreLink`), but we can just have a conditional to show the app badge like how you had before, `<% if (iosURL) { %> // image` and also include the storybooks to show one or the other, I think that'd be awesome. Sorry to ask you to change something I suggested you change in the first place.